src/Symfony/Component/Notifier/Exception/FlashMessageMappingException.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Exception;
+
+/**
+ * @author Ben Roberts <ben@headsnet.com>
+ */
+class FlashMessageMappingException extends LogicException
```

Comment:
We should add the word "importance" to the exception name somehow

Reply:
Agreed - will push changes in a moment.

Reply:
Renamed to FlashMessageImportanceMapperException